Ls_11/style.css

```diff
@@ -40,10 +40,10 @@ nav a{
     height: 60 px;
     line-height: 60px;
 }
-nav a:hover{
+nav a :hover{
     background:#191a19;
 }
-nav a:nth-child(2) {
+nav a :nth-child(2) {
```

Comment:
why you added these spaces?
not needed

Reply:
this for lesson 11. In lesson 14 is good)